packages/main/src/MenuItem.js

```diff
@@ -167,6 +189,10 @@ class MenuItem extends UI5Element {
 		return !!Object.keys(this._subMenu).length;
 	}
 
+	get _additionalText() {
+		return this.hasChildren || this.hideAdditionalText ? "" : this.additionalText;
```

Comment:
I think it's good to document above in the API (f.e. with a note), that the `additionalText` won't be displayed if the item has nested items (this.hasChildren).

Reply:
yes, you're right. Added note in the API.